File: analytics/src/main/java/io/appmetrica/analytics/impl/component/session/SessionArguments.java
package io.appmetrica.analytics.impl.component.session;

public class SessionArguments {

    public final long creationElapsedRealtime;
    public final long creationTimestamp;

    public SessionArguments(long creationElapsedRealtime, long creationTimestamp) {
        this.creationElapsedRealtime = creationElapsedRealtime;
        this.creationTimestamp = creationTimestamp;
    }
}

File: analytics/src/main/java/io/appmetrica/analytics/impl/component/session/SessionArguments.kt
package io.appmetrica.analytics.impl.component.session

internal class SessionArguments @JvmOverloads constructor(
    val creationElapsedRealtime: Long,
    val creationTimestamp: Long,
    val sessionRequestParams: SessionRequestParams? = null
) {
    override fun toString(): String {
        return "SessionArguments(" +
            "creationElapsedRealtime=$creationElapsedRealtime, " +
            "creationTimestamp=$creationTimestamp, " +
            "sessionRequestParams=$sessionRequestParams" +
            ")"
    }
}
